doc/.known-issues/doc/kerneldoc.conf

```diff
@@ -6,3 +6,5 @@
 ^Invalid C declaration:.*
 ^  long virtio_.*
 ^[- \t]*\^
+^(?P<filename>[-._/\w]+/acrn-kernel/drivers/gpu/drm/i915/gvt/mpt.h):(?P<lineno>[0-9]+): warning: Function parameter .* not described in .*
```

Comment:
The right fix is to fix the incorrect code kerneldoc comments, not to hide the error message.  Remove these two additional lines (they're incorrectly done anyway).

Reply:
Hi @dbkinder,
It's in upstream i915 driver. Previously, we keep an offline patch in acrn-kernel(5.4 branch) to pass this doc build. Now, after switching to 5.10 base, PKT kernel doesn't accept non-upstream patch anymore. 
Of course, we can upstream this doc fix in i915 driver. However, it's another repo (kernel) business. While our acrn-hypervisor will be blocked until the patch is accepted by Linux community and backported.

Such dependency is not good. This change can remove the dependency.

Reply:
Can we make the comment changes in the mpt.h file in the acrn-kernel repo we maintain? This is what was done in the past for our API document generation.  The change to the known-issues pattern hides the error message, but the API documentation won't be generated correctly.

Reply:
I can do that if the doc build using acrn-kernel.